### DISCUSSION TITLE
Validate integrity of the backups -

### DISCUSSION THREAD

**Discussion by CodeWithCJ** [garethgeorge/backrest#621] [Q&A]
TLDR: Backup shows 232GB on Backblaze after multiple failed retries; Daily chart only shows 41GB+. Ran checks, tested random files, all seems fine—can I trust it’s complete with 60K+ files properly?

Full post: 
I started the backup on 12/14. The chart below max shows 41GB on 12/15. My total backup is 232GB as shown in the second screenshot.  Backup job was running for couple of days to complete. It often failed and I had to run backup multiple times to finally complete. 
Is the backup properly completed as the size on the daily image (first screenshot) and second screenshot are not matching. 

I tried running CHECK option and PRUNE. Both completed successfully. Also tried to manually download only few files randomly to test, it worked fine. Backup size on backblaze web portal also shows 232GB. Is the backup completed properly? Don't want to get into trouble when I need to restore? As its close to 60K+ images & videos, hard to test if every single one of them was backed up correctly or messed up. 

Should I delete everything and start from scratch to avoid issues in the future? 

![image](https://github.com/user-attachments/assets/ecbd52d4-f86e-4691-895a-1895952c80dc)

![image](https://github.com/user-attachments/assets/9574c6f9-e9aa-47a9-a492-8a660bc6dbd9)

![image](https://github.com/user-attachments/assets/4b14bc93-0ce1-4896-a3fa-c04c5b586781)

**Comment by garethgeorge** [comment]
If your check operation returns successful (especially with a random sampling of the files verified using the check % option) you can trust that your data is uploaded with high confidence. 

If you had some trouble getting all of your files uploaded, it's likely that backrest never saw the successful summary that restic prints at the end of some of those attempts and didn't correctly capture the data they uploaded. The summary dashboard is a useful tool but, especially when you've had a number of backups partially fail, its data can become incorrect.

There is another repo summary tool available on the stats panel of the repo summary view. Click "compute stats now" and wait for it to populate, numbers on that panel should be authoritative re: space used, etc.

<img width="1675" alt="image" src="https://github.com/user-attachments/assets/0fd195b5-7c3c-44d5-9979-c989e371a408" />
